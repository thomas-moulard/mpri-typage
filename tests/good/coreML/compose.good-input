val compose f g = fun x. f (g x)
