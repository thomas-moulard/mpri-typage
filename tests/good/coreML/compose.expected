compose = fun f.fun g.fun x.f (g x)

compose : forall 'a 'b 'c.(('c -> 'b) -> (('a -> 'c) -> ('a -> 'b)))
